src/testcafe-runner.ts

```diff
@@ -313,15 +340,18 @@ async function runTestCafe (tcCommandLine, projectPath) {
   return { startTime, endTime, hasPassed };
 }
 
-async function run (nodeBin, runCfgPath, suiteName) {
+async function run (nodeBin: string, runCfgPath: string, suiteName: string) {
   const preExecTimeout = 300;
 
   const cfg = await prepareConfiguration(nodeBin, runCfgPath, suiteName);
   if (!cfg) {
     return false;
   }
+  const suite = {
+    preExec: (cfg.suite as any).preExec
```

Comment:
Would be fantastic if you could introduce the saucectl Suite definition of testcafe as a type 🙏